paper-slider.html

```diff
@@ -198,24 +198,22 @@
         margin: 10px;
         width: calc(100% - 20px);
         height: calc(100% - 20px);
-        border-radius: 50%;
         background-color: var(--paper-slider-knob-color, --google-blue-700);
+        border: 1px solid var(--paper-slider-knob-color, --google-blue-700);
+        border-radius: 50%;
 
         -moz-box-sizing: border-box;
         box-sizing: border-box;
 
-        transition-property: margin, height, width, background-color, border;
```

Comment:
wow, didn't realize this element was causing layout. That might explain https://github.com/PolymerElements/paper-slider/pull/87